### ISSUE TITLE
CSS for Phrasing is not being included in the asset pipeline

### ISSUE THREAD

**Issue by ryanple** [infinum/phrasing#43]
This is probably moreso project-specific for me; but I'm sure others maybe have run into this. I'm using Sass and no matter where I put phrasing in the order of application.scss, the CSS isn't included. If I create a phrasing.scss and include it at the bottom of my application.scss, all of the CSS will overwrite my existing CSS and change the fonts and sizes when a user is logged in to edit. Is there anyway around this that I can get the floating edit-box on the bottom right, without having all my other CSS overwritten? I feel like this would confuse my client if I manually created a phrasing.scss file with all the custom assets/stylesheets/* from phrasing, but I also want the sweet edit-box in the bottom right. Thanks :)
![image](https://cloud.githubusercontent.com/assets/6797308/7958017/8b03ce4e-09b9-11e5-9477-434509a65848.png)

**Comment by DamirSvrtan** [comment]
Can you show me code examples of what you've tried so far?

**Comment by ryanple** [comment]
Hey Damir, thanks for responding.

I'm using entirely defaults based on the instructions. I'm using a wrap-bootstrap theme with the project. The css files from the theme are overriding phrasing CSS.

What I tried doing was taking all the CSS from the phrasing repository and creating my own phrasing.scss under app/assets/stylesheets, but then it overrides the appearance of the theme.

I guess my main question is, what is the necessary css for the floating status window?

Sorry if this doesn't make much sense! I just want the theme's css and phrasing css to get along. lol.

**Comment by ryanple** [comment]
I got it figured out.

I created a file app/assets/stylsheets/phrasings.scss (yes with an extra S), and pasted in the CSS from phrasing.css.scss, as well as phrasing_edit_mode_bubble.css.scss, and it looks perfectly how I want it to... edit content bubble is dynamic on the lower right without affecting appearance on the page.

Although very "hackey", it's functional. :)

Thanks for the awesome gem, Damir! You guys are great.
